Description: Don't show printers and region panels in Unity until we're
 ready for them
Forwarded: not-needed
Author: Jeremy Bicha <jbicha@ubuntu.com>
Index: gnome-control-center-3.6.3/panels/printers/gnome-printers-panel.desktop.in.in
===================================================================
--- gnome-control-center-3.6.3.orig/panels/printers/gnome-printers-panel.desktop.in.in	2012-11-05 03:38:42.000000000 -0500
+++ gnome-control-center-3.6.3/panels/printers/gnome-printers-panel.desktop.in.in	2012-11-14 19:41:32.183905747 -0500
@@ -8,7 +8,7 @@
 StartupNotify=true
 # The X-GNOME-Settings-Panel is necessary to show in the main shell UI
 Categories=GNOME;GTK;Settings;HardwareSettings;X-GNOME-Settings-Panel;
-OnlyShowIn=GNOME;Unity;
+OnlyShowIn=GNOME;
 X-GNOME-Settings-Panel=printers
 # Translators: those are keywords for the printing control-center panel
 _Keywords=Printer;Queue;Print;Paper;Ink;Toner;
Index: gnome-control-center-3.6.3/panels/region/gnome-region-panel.desktop.in.in
===================================================================
--- gnome-control-center-3.6.3.orig/panels/region/gnome-region-panel.desktop.in.in	2012-11-05 03:38:42.000000000 -0500
+++ gnome-control-center-3.6.3/panels/region/gnome-region-panel.desktop.in.in	2012-11-14 19:41:32.183905747 -0500
@@ -7,7 +7,7 @@
 Type=Application
 StartupNotify=true
 Categories=GNOME;GTK;Settings;DesktopSettings;X-GNOME-Settings-Panel;X-GNOME-PersonalSettings
-OnlyShowIn=GNOME;Unity;
+OnlyShowIn=GNOME;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-control-center
 X-GNOME-Bugzilla-Component=region
Index: gnome-control-center-3.6.3/panels/background/gnome-background-panel.desktop.in.in
===================================================================
--- gnome-control-center-3.6.3.orig/panels/background/gnome-background-panel.desktop.in.in	2012-11-05 03:38:42.000000000 -0500
+++ gnome-control-center-3.6.3/panels/background/gnome-background-panel.desktop.in.in	2012-11-14 19:41:56.771904719 -0500
@@ -7,7 +7,7 @@
 Type=Application
 StartupNotify=true
 Categories=GNOME;GTK;Settings;DesktopSettings;X-GNOME-Settings-Panel;X-GNOME-PersonalSettings
-OnlyShowIn=GNOME;
+OnlyShowIn=GNOME;Unity;
 X-GNOME-Settings-Panel=background
 # Translators: those are keywords for the background control-center panel
 _Keywords=Wallpaper;Screen;Desktop;
